scene/gui/line_edit.cpp

```diff
@@ -1774,10 +1774,10 @@ Size2 LineEdit::get_minimum_size() const {
 	// Minimum size of text.
 	float em_space_size = font->get_char_size('M', font_size).x;
 	min_size.width = theme_cache.minimum_character_width * em_space_size;
-
+	
```

Comment:
You added a tab on this empty line, which conflicts with our style guide. See https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#using-clang-format-locally